src/user/user.spec.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env jest */
+
+import { omit } from "lodash";
+import { testConnection, xo } from "../util";
+
+const simpleUser = {
+  email: "wayne3@vates.fr",
+  password: "batman",
+};
+
+describe("user", () => {
+  afterAll(async () => {
+    await xo.deleteAllUsers();
+  });
+
+  describe(".create()", () => {
+    beforeAll(async () => {
+      await xo.createUser(simpleUser);
+    });
```

Comment:
This pre-condition is only used for a single test, inline it.

Reply:
If I put this pre-condition in "failed test" only to create two users with the same email, I must remove this test from the describe.each.

So, we should have : 
- describe.each("successfully")
- describe.each("failed")
- and another test("failed") with this one.

Is that what you expect ?